diff -ruN ../jsmenubar-orig/jsdomenubar.js ./jsdomenubar.js
--- ../jsmenubar-orig/jsdomenubar.js	2005-02-12 00:10:56.000000000 +0100
+++ ./jsdomenubar.js	2006-10-18 11:11:56.000000000 +0200
@@ -581,6 +581,9 @@
   if (len > 5 && typeof(arguments[5]) == "number" && arguments[5] > 0) {
     menuBarElm.style.height = arguments[5] + px;
   }
+  
+  menuBarElm.style.width = "99%"
+  
   menuBarElm.style.left = "0px";
   menuBarElm.style.top = "0px";
   if (ie50) {
